### PR TITLE
[storage] Unexposes models from aio

### LINKED ISSUE #7864: [Storage] Un-expose models from aio

### PR DESCRIPTION
**Pull request by kristapratico**
Resolves #7864 #7863

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by kristapratico** [comment]
> Are you planning to hide some of the sync models in the same PR? or is there a different issue to track it?

@johanste  @annatisch

**Comment by annatisch** [comment]
I think the only sync models to hide are the Paged ones - is that correct?

File: sdk/storage/azure-storage-blob/HISTORY.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 - `set_container_access_policy` has required parameter `signed_identifiers`.
 - NoRetry policy has been removed. Use keyword argument `retry_total=0` for no retries. 
+- Now only the async clients `BlobServiceClient`, `ContainerClient`, `BlobClient`, and `LeaseClient` can be imported
```
Comment (johanste) [inline sdk/storage/azure-storage-blob/HISTORY.md]: Change phrasing to something like:

"Removed types that were accidentally exposed from two modules. Only BlobServiceClient, ContainerClient, BlocBlient and LeaseClient should be imported from azure.storage.blob.aio".

?

**Pull request merged**